Packs/Smokescreen_IllusionBLACK/IncidentFields/incidentfield-IllusionBLACK_Events.json

```diff
@@ -0,0 +1,81 @@
+{
+	"associatedToAll": false,
+	"associatedTypes": [
+		"IllusionBLACK"
+	],
+	"breachScript": "",
+	"caseInsensitive": true,
+	"cliName": "illusionblackevents",
+	"closeForm": false,
+	"columns": [
+		{
+			"displayName": "Column Header 1",
```

Comment:
Why is the displayname on all of those is `Column Header 1` ?

Reply:
It is kept blank for now. It is never showed in any layout. To accomodate lists, the type was table so that I can use it in playbooks.